.isort.cfg

```diff
@@ -1,5 +1,5 @@
 [settings]
-known_third_party = attr,dill,jinja2,migrate,parse,pathos,pkg_resources,plumbum,psutil,pygit2,pygtrie,pyparsing,pytest,rich,setuptools,six,sqlalchemy,typing_extensions,yaml
```

Comment:
uh, that is weird.

My local python is python3.8, with pre-commit enabled. isort will ran with python3.8.
In the code, I select Protocol from type_extensions, when python ist <= 3.8, otherwise we use the version
from typing.

So.... from isort's perspective: there is no typing_extensions module.

... :(